### PR TITLE
[cherry-pick] [PLUGIN-1694] Amazon S3 Batch Source Plugin Release/1.17.4 

### PR DESCRIPTION
**Pull request by sgarg-CS**
CDAP: v6.7
JIRA: [PLUGIN-1694](https://cdap.atlassian.net/browse/PLUGIN-1694)

Original `develop` PR: https://github.com/data-integrations/amazon-s3-plugins/pull/185

[PLUGIN-1694]: https://cdap.atlassian.net/browse/PLUGIN-1694?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
   <groupId>io.cdap.plugin</groupId>
   <artifactId>amazon-s3-plugins</artifactId>
-  <version>1.17.3</version>
+  <version>1.17.4</version>
```
Comment (MrRahulSharma) [inline pom.xml]: Please take this change separately.

Comment (itsankit-google) [reply pom.xml]: +1, we should change it to `1.17.4-SNAPSHOT` as part of this PR.

**Review by itsankit-google — COMMENTED** [review]
Please also add the original develop PR in the description.

File: pom.xml (hunk 1 above)
Comment (sgarg-CS) [reply pom.xml]: Updated to `1.17.4-SNAPSHOT` as part of this PR. 79e1df818f856438b79b4431b49d12d8ceb8811c
Will raise a separate PR for removing the `-SNAPSHOT`

**Comment by sgarg-CS** [comment]
> Please also add the original develop PR in the description.

Added

**Review by MrRahulSharma — APPROVED** [review]

**Comment by MrRahulSharma** [comment]
Please investigate the build failure in this change before submission.

**Review by MrRahulSharma — APPROVED** [review]

**Pull request merged**